playground/backend/internal/preparators/java_preparators_test.go

```diff
@@ -37,7 +39,7 @@ func Test_replace(t *testing.T) {
 	if err != nil {
 		panic(err)
 	}
-	lc, _ := fs_tool.NewLifeCycle(pb.Sdk_SDK_JAVA, uuid.New(), filepath.Join(path, "temp"))
+	lc, _ := fs_tool.NewLifeCycle(pb.Sdk_SDK_JAVA, uuid.New(), filepath.Join(path, "temp", pipelinesFolder))
```

Comment:
It is not necessary. `fs_tool.NewLifeCycle` receive any directory as the last argument and it doesn't matter if it will be `temp/executable_files` or just `executable_files`.